dist/tools/ethos/setup_network.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+create_tap() {
+    ip tuntap add ${TAP} mode tap user ${_USER}
+    sysctl -w net.ipv6.conf.${TAP}.forwarding=1
+    sysctl -w net.ipv6.conf.${TAP}.accept_ra=0
+    ip link set ${TAP} up
+    ip a a fe80::1/64 dev ${TAP}
+    ip a a fd00:dead:beef::1/128 dev lo
+    ip route add ${PREFIX} via fe80::2 dev ${TAP}
+}
+
+remove_tap() {
+    ip tuntap del ${TAP} mode tap
+}
+
+cleanup() {
+    echo "Cleaning up..."
+    remove_tap
+    ip a d fd00:dead:beef::1/128 dev lo
+    trap "" INT QUIT TERM EXIT
+}
+
+start_uhcpd() {
+    ${UHCPD} ${TAP} ${PREFIX}
+}
+
+TAP=$1
+PREFIX=$2
+_USER=$3
+UHCPD="../uhcpd/bin/uhcpd"
```

Comment:
This does not work if its not called from `/dist/tools/ethos/`, change for `UHCPD="$(readlink -f $(dirname $0)"/../uhcpd/bin")/uhcpd"`?

Reply:
good point. changed to ```: ${UHCPD:="$(readlink -f $(dirname $0)"/../uhcpd/bin")/uhcpd"}``` so it can still be overridden.